blinky.md

```diff
@@ -58,6 +52,8 @@ to run your code in Tessel's RAM.
 
 **Look at your Tessel!** The blue and green LEDs on your Tessel's LED panel should blink back and forth.
 
+![](http://i.imgur.com/kZIZNcL.gif)
```

Comment:
Can you format this to match the left/right of the Tessel piece above or similar?
Actually messed around with this, I'll PR your branch